spec/mailers/referee_mailer_spec.rb

```diff
@@ -19,65 +19,101 @@
+      )
 
-      expect(email.body).to include('for example about their academic record')
-    end
+      it 'adds additional guidance for academic references' do
+        reference.referee_type = :academic
+
+        expect(email.body).to include('for example about their academic record')
+      end
 
-    it 'adds additional guidance for character references' do
-      reference.referee_type = :character
+      it 'adds additional guidance for character references' do
+        reference.referee_type = :character
 
-      expect(email.body).to include('for example about activities you have done together')
+        expect(email.body).to include('for example about activities you have done together')
+      end
+
+      %i[professional school_based].each do |referee_type|
+        it "adds additional guidance for #{referee_type} references" do
+          reference.referee_type = referee_type
+
+          expect(email.body).to include('for example about their role and responsibilities at work')
+        end
+      end
     end
 
-    %i[professional school_based].each do |referee_type|
-      it "adds additional guidance for #{referee_type} references" do
-        reference.referee_type = referee_type
+    context 'with confidentiality feature flag inactive' do
+      before { FeatureFlag.deactivate(:show_reference_confidentiality_status) }
+
+      let(:email) { mailer.reference_request_email(reference) }
+      let(:application_choices) { [create(:application_choice, :accepted, course_option:)] }
 
-        expect(email.body).to include('for example about their role and responsibilities at work')
+      it 'does not include the confidentiality statement' do
+        expect(email.body).not_to include(
+          'You can choose whether Elliot will be able to see your reference or if it should be kept confidential.',
+        )
       end
     end
   end
 
   describe 'Send chasing reference email' do
-    let(:email) { mailer.reference_request_chaser_email(application_form, reference) }
-    let(:application_choices) { [create(:application_choice, :accepted, course_option:)] }
+    context 'with confidentiality feature flag active' do
+      before { FeatureFlag.activate(:show_reference_confidentiality_status) }
 
-    it 'sends an email with a link to the reference form' do
-      expect(email.body).to include('/reference?token=raw_token')
+      let(:email) { mailer.reference_request_chaser_email(application_form, reference) }
+      let(:application_choices) { [create(:application_choice, :accepted, course_option:)] }
+
+      it 'sends an email with a link to the reference form' do
+        expect(email.body).to include('/reference?token=raw_token')
+      end
+
+      it_behaves_like(
+        'a mail with subject and content',
+        I18n.t('referee_mailer.reference_request.subject', candidate_name: 'Elliot Alderson'),
```

Comment:
Not a blocker, but shall we use the actual subject text rather than the translation reference?

Reply:
Sure. I didn't change these but happy to update. 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10291/commits/ce8ce0008b88e230d3d7c790ae84e8a49453b5a6